### PR TITLE
Add `pre-release-testing` document describing the QA testing process

### LINKED ISSUE #192: Document QA process for merging into `production`
It would be useful to have a list of things to check during QA. This makes QA easier/faster and also avoids that we forget checking certain things. The document doesn't need to be super formal, but should contain a somewhat complete list of things to check.

### PR DESCRIPTION
**Pull request by LukasKalbertodt**
Closes #192 

This is just a rough guideline that we can follow while testing. This should make things easier for us since we don't have to think of everything ourselves. This doesn't mean that we always have to follow the steps exactly.

Does this contain all important things? Does it specify too much? Anything else I should change?

PS: I mentioned a few checks already that are currently still buggy (i.e. disabled links while recording).

### CODE REVIEW

File: doc/pre-release-testing.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+### Testing Steps
+
+#### UI
+
+- **Logo and "Opencast Studio"** are visible?
+- Make sure **only supported options** are shown:
+    - Mobile: only "Share webcam", not "Share desktop"
+    - Desktop: should show both, "webcam" and "desktop"
+- Make sure there are **no scrollbars** anywhere.
+- Does page **"Legal notices"** exist and is easily reachable?
```
Comment (lkiesow) [inline doc/pre-release-testing.md]: Will be merged into about anyway, so…
```suggestion
```

File: doc/pre-release-testing.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+- Make sure all **links are disabled** (TODO: this still needs to be implemented)
+- Pause and then resume
+- Stop the recording
+- Make sure the "production details" dialog pops up
+- Check fields "Title" and "Presenter"
+- Is the **media preview working**?
+- Check buttons "Upload to Opencast", "Save Media" and "Discard"
+    - Do they have useful colors (i.e. red for "discard")?
+- Click "Upload to Opencast"
+    - Make sure that the video arrived on the OpenCast side correctly
```
Comment (lkiesow) [inline doc/pre-release-testing.md]:
```suggestion
    - Make sure that the video was correctly uploaded to Opencast
```
Opencast does not include a capital c.

**Comment by LukasKalbertodt** [comment]
@lkiesow Or do you want me to fixup the original commit instead of creating a new one? I just used the GitHub feature because it seemed really easy right now ^_^

**Comment by lkiesow** [comment]
I think that's the answer :-)

**Pull request merged**